docs/release-notes.7.0.md

```diff
@@ -1,5 +1,8 @@
 # Qooxdoo Release Notes
 
+## Fixes for v7.7.2
+For a full list of changes see https://github.com/qooxdoo/qooxdoo/commits/master?branch=master&qualified_name=refs%2Fheads%2Fmaster&since=2024-05-13&until=2024-19-07
```

Comment:
Should be 
https://github.com/qooxdoo/qooxdoo/commits/master?branch=master&qualified_name=refs%2Fheads%2Fmaster&since=2024-05-03&until=2024-22-07

Reply:
Pay attention: You need a github and an npm key.

If you close the PR i can release the new version directly

Reply:
Thank you, I fixed dates but still need approvals to merge it in the same day which pointed out in date interval. I don't have npm key. And I have to access to the our repository to publish don't I?